messaging/copr_messaging/schema.py

```diff
@@ -46,6 +46,14 @@ def __str__(self):
             self.status,
         )
 
+    @property
+    def agent_name(self):
+        """The username who caused the action that generated this message."""
+        if self.status == "canceled":
+            return self.submitter
+        # The other statuses are not caused by the user who started the build
```

Comment:
Strictly speaking, anyone who has build or admin access to the project should be able to cancel the build.  I don't think we have the needed info here.

Reply:
Indeed, would it be possible to add that info to the message body?

Reply:
I think so, but this seems like a separate issue (the asynchronous cancelation on the remote machine is relatively complicated logic).

Reply:
So, does it make sense to mix up the submitter with the person who canceled the build?  If yes, I think this PR is good to go. Otherwise, we should do something about it first, ideas?

Reply:
Fixed! Thanks.

Reply:
To be safe I didn't assume the submitter was the one who canceled the build. This may create notifications for people who didn't want to get notified of their own actions, but that's a lesser evil I think.